### ISSUE TITLE
Public channel with password cannot be joined by user

### ISSUE THREAD

**Issue by jy2008** [RocketChat/Rocket.Chat#6041]
Public channel with password cannot be joined by user

**Comment by rodrigok** [comment]
1. `Administration > Accounts > Registration` their you have 2 options `Email Verification` will allow user login only after the email verification and `Use DNS Domain Check` that verify if the domain of the email exists.

2. `Administration > Mailer`

3. Is this one or two problems? Can you open a new issue for that? Then I'll investigate.

4. You are right, I'll change the title of this issue to reflect this problem and will investigate ASAP.

Thanks

**Comment by SeanPackham** [comment]
Tested on 0.61.2 and I can join a public channel which is restricted by a password if I supply the correct password.
